components/feature/prompts/src/main/java/mozilla/components/feature/prompts/dialog/PromptDialogFragment.kt

```diff
@@ -28,6 +29,8 @@ internal abstract class PromptDialogFragment : DialogFragment() {
 
 internal interface Prompter {
 
+    val loginsDelegate: LoginsDelegate
```

Comment:
We need access to the LoginsDelegate to communicate with Logins storage, but can't pass it to the fragment. This is what we came up, but let us know if you have other suggestions!